### PR TITLE
added Velocity Support

### PR DESCRIPTION
**Pull request by AzraAnimating**
Modified the Bungee api's code so it'll work natively with Velocity Proxy, left most of the original code in place where possible.

(I would maintain this module if needed)

### CODE REVIEW

File: blcmodapivelocity/src/main/java/net/badlion/blcmodapivelocity/BlcModApiVelocity.java
Comment (PSNRigner) [inline blcmodapivelocity/src/main/java/net/badlion/blcmodapivelocity/BlcModApiVelocity.java:55]: Missing a letter

File: blcmodapivelocity/src/main/java/net/badlion/blcmodapivelocity/BlcModApiVelocity.java
Comment (PSNRigner) [inline blcmodapivelocity/src/main/java/net/badlion/blcmodapivelocity/BlcModApiVelocity.java:83]: Missing `this.`

**Review by PSNRigner — CHANGES_REQUESTED** [review]
Hey, thank your for the PR, that would definitely be interesting to add.

I still need to ask for a second review from my boss to make sure it's all good.

I marked a few details on the PR, unfortunately at Badlion we have a very strict coding style and there are just 2 small lines to change 😉

**Comment by AzraAnimating** [comment]
Should be fixed :)

**Comment by PSNRigner** [comment]
Hey, I'm sorry I never looked into it. Will do during the week, we completely remade this plugin recently (see the development branch), we'll add support for Velocity as well 😀

**Comment by james090500** [comment]
That'll be amazing!

**Comment by PSNRigner** [comment]
Velocity support has been added into the upcoming rework of the ModApi. Check the development branch!

**Pull request closed without merging**
